packages/ckeditor5-emoji/theme/emojipicker.css

```diff
@@ -30,3 +30,8 @@
 	margin-right: -1px;
 	outline: solid 1px hsla(0, 0%, 100%, .5);
 }
+
+/* See: https://github.com/ckeditor/ckeditor5/issues/17964 */
+div.ck.ck-balloon-panel:has( .ck.ck-emoji ) {
+	z-index: calc( var( --ck-z-dialog ) + 1 );
```

Comment:
Comments archive uses `--ck-z-panel` for the `z-index`. Could you check if after replacing the variable it still works fine?

Reply:
It does not work. See:

https://github.com/ckeditor/ckeditor5/blob/26bf48f62284cffdb1079083c509d3a7d091b386/packages/ckeditor5-ui/theme/globals/_zindex.css#L6-L10